indico/web/errors.py

```diff
@@ -27,6 +27,10 @@
 def render_error(exc, title, message, code, standalone=False):
     _handle_error_cors()
     _save_error(exc, title, message)
+    
+    if isinstance(exc, UnicodeDecodeError):
+        message = "You need to save your CSV file using the UTF-8 encoding."
```

Comment:
This doesn't belong here. If you want to catch such errors, handle them in the place where they originate...

Reply:
I have a few questions: 

1. How do you lint the code? 
2. `indico db prepare` is not working for me is there any proper documentation for this setting up the project?

Reply:
1. `ruff check`
2. https://docs.getindico.io/en/stable/installation/development/

Reply:
thank you